packages/react/src/themes/teams-dark/components/Tree/treeTitleVariables.ts

```diff
@@ -0,0 +1,7 @@
+import { TreeTitleVariables } from '../../../teams/components/Tree/treeTitleVariables'
+
+export default (siteVars: any): TreeTitleVariables => {
+  return {
+    defaultColor: siteVars.colors.white,
+  }
+}
```

Comment:
Can be removed:
https://github.com/stardust-ui/react/pull/1779/files#r319038491